kythe/typescript/indexer.ts

```diff
@@ -363,16 +374,43 @@ class Vistor {
   }
 
   /** getSymbolName computes the VName (and signature) of a ts.Symbol. */
-  getSymbolName(sym: ts.Symbol, ns: TSNamespace): VName {
+  getSymbolName(sym: ts.Symbol, ns: TSNamespace, context?: ts.Node): VName {
```

Comment:
Can you define (in the comment above) what context is?

Reply:
Yep, done. I actually had another version I forgot to merge, but I did now so some of this is expired.